src/test/java/org/broadinstitute/hellbender/utils/mcmc/SliceSamplerUnitTest.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.utils.mcmc;
+
+import com.google.common.primitives.Doubles;
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.random.RandomGenerator;
+import org.apache.commons.math3.random.RandomGeneratorFactory;
+import org.apache.commons.math3.stat.descriptive.moment.Mean;
+import org.apache.commons.math3.stat.descriptive.moment.StandardDeviation;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Random;
+import java.util.function.Function;
+
+
+/**
+ * Unit tests for {@link SliceSampler}.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+public final class SliceSamplerUnitTest {
+    private static final int RANDOM_SEED = 42;
+    private static final RandomGenerator rng =
+            RandomGeneratorFactory.createRandomGenerator(new Random(RANDOM_SEED));
+
+    private static double relativeError(final double x, final double xTrue) {
+        return Math.abs((x - xTrue) / xTrue);
+    }
+
+    /**
+     * Test slice sampling of a normal distribution.  Checks that input mean and standard deviation are recovered
+     * by 20000 samples to a relative error of 0.05% and 2%, respectively.
+     */
+    @Test
+    public void testSliceSamplingOfNormalDistribution() {
+        final double mean = 5.;
+        final double standardDeviation = 0.75;
+        final NormalDistribution normalDistribution = new NormalDistribution(mean, standardDeviation);
+        final Function<Double, Double> normalLogPDF = x -> normalDistribution.logDensity(x);
+
+        final double xInitial = 4.5;
```

Comment:
Make xInitial several standard deviations away from the mean because a slice sampler should be robust against bad initial guesses.  To be honest I can't think of any coding error that would fail in this way without failing in general, but it just seems right to stack the test against the code.